brainframe/api/stubs/zone_statuses.py

```diff
@@ -66,15 +66,15 @@ def zone_status_iterator():
                     raise bf_errors.ServerNotReadyError(message)
 
                 if packet == b'':
-                    continue
-
-                # Parse the line
-                zone_statuses_dict = json.loads(packet)
-
-                processed = {
-                    int(s_id): {key: bf_codecs.ZoneStatus.from_dict(val)
-                                for key, val in statuses.items()}
-                    for s_id, statuses in zone_statuses_dict.items()}
+                    processed={}
```

Comment:
I think this change will cause downstream problems. Returning `{}` has semantic meaning: It means there are 0 streams. 

Feel free to merge it, but consider instead raising an exception, or returning a sentinal value such as None.

Reply:
@apockill In some scenarios the streams will stop and be removed after running a while, so the {} is expected for these scenarios. Maybe when there are streams running, an empty string or {} will return occasionally. If so an incompatible problem may happen. Anyway, with this mechanism, the API can tell the application what happened and the application can make a decision according to the return values.